README.md

```diff
@@ -23,7 +23,7 @@ For more details on the steps above, refer to [Setting up O3DE from GitHub](http
     ```
 2. Register gems with this command:
     ```
-    scripts\o3de.bat register -gp <your path to o3de-extras>/<gem name>
+    scripts\o3de.bat register -gp <your path to o3de-extras>/Gems/<gem name>
```

Comment:
The register command supports recursive registration of all gems under a path through the [-agp/--all-gems-path](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/register.py#L870-L871) option.

The enable-gem command supports explicit activation of all gems under a path using the [-agp/--all-gem-paths](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/enable_gem.py#L196-L197) option.

I think we should mention that as an alternative step for registering all gems in the o3de-extra repo vs registering individual gems

Reply:
Sure, can add that in there